src/components/apps/launch/InputSelector.js

```diff
@@ -97,17 +97,12 @@ const InputSelector = ({
     const { t } = useTranslation("launch");
     const { t: prefI18n } = useTranslation("preferences");
     const { setFieldValue } = form;
-    const [bootstrapInfo, setBootstrapInfo] = useBootstrapInfo();
+    const bootstrapInfo = useBootstrapInfo()[0];
```

Comment:
No need to change this (makes sense as-is), but this also works:

```suggestion
    const [bootstrapInfo] = useBootstrapInfo();
```